lib/1.4/dml-builtins.dml

```diff
@@ -2929,7 +2929,7 @@ template register is (_conf_attribute, get, set, shown_desc,
         return this.val & enabled_bits;
     }
 
-    shared method read_register(uint64 enabled_bytes, void *aux)-> (uint64) default {
+    shared method _base_read_register(uint64 enabled_bytes, void *aux)-> (uint64) {
```

Comment:
Would not hurt with an explanatory comment, e.g. it's internal/subject to change, stand-in for a hopefully-upcoming feature, see SIMICS-NNNN, etc

Reply:
SIMICS-22264, that is

Reply:
Done.